peachjam/forms.py

```diff
@@ -112,7 +112,7 @@ def filter_queryset(self, queryset, exclude=None):
         registries = self.params.getlist("registries")
 
         # Order by date descending initially
-        queryset = queryset.order_by("-date")
+        queryset = queryset.order_by("-date", "title")
```

Comment:
Nice catch -- my guess is that ordering of rows with the same date is not guaranteed, and so can change. By ordering by date and then title, we guarantee a consistent ordering.

Reply:
https://stackoverflow.com/questions/67343360/is-order-by-stable-for-several-rows-with-the-same-key-values